src/components/search-results/index.js

```diff
@@ -75,9 +76,9 @@ class SearchResults extends Component {
             <VisbilitySensor key={index} onChange={(isVisible) => this.handleVisibility(isVisible, item)}>
               <div className='gridItem'>
                 <div onClick={() => removeTile(item.id)}>
-                  <img className='removeTileButton' src='../../src/assets/cancel.svg' alt='cancelled' />
+                  <img className='removeTileButton' src={removeTileButton} alt='cancelled' />
                 </div>
-                <div key={index} onClick={() => changeRoute(`/hotel/${item.url}`)}>
+                <div key={index} onClick={() => changeRoute(`/hotel/${resultId}/`)}>
```

Comment:
oops. this will no longer work... it should be `item.url` ... @Kumjami please confirm.